docker_images_list.md

```diff
@@ -16,7 +16,7 @@ Take ChatQnA for example. ChatQnA is a chatbot application service based on the
 | [opea/chatqna-conversation-ui](https://hub.docker.com/r/opea/chatqna-conversation-ui)                       | [Link](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/ui/docker/Dockerfile.react)           | Chatqna React UI. Facilitates interaction with users, enabling chat-based Q&A with conversation history stored in the browser's local storage.                                                                                       | [Link](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/ui/react/README.md)           |
 | [opea/chatqna-ui](https://hub.docker.com/r/opea/chatqna-ui)                                                 | [Link](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/ui/docker/Dockerfile)                 | Chatqna UI entry. Facilitates interaction with users to answer questions                                                                                                                                                             | [Link](https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/ui/svelte/README.md)          |
 | [opea/codegen](https://hub.docker.com/r/opea/codegen)                                                       | [Link](https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/Dockerfile)                           | Codegen gateway. Provides automatic creation of source code from high-level representations                                                                                                                                          | [Link](https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/README.md)                    |
-| [opea/codegen-gradio-ui]()                                                                                  | [Link](https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/ui/docker/Dockerfile.gradio)          | Codegen Gradio UI entry. Interact with users to generate source code by providing high-level descriptions or inputs.                                                                                                                 |                                                                                                      |
+| [opea/codegen-gradio-ui]()                                                                                  | [Link](https://github.com/opea-project/GenAIExamples/blob/main/CodeGen/ui/docker/Dockerfile.gradio)          | Codegen Gradio UI entry. Interact with users to generate source code by providing high-level descriptions or inputs.                                                                                                                 | [Link](CodeGen/ui/gradio/README.md)                                                                  |     |
```

Comment:
I think for the link we need the full URL because the relative URL won't resolve properly by Dockerhub.